notebooks/text_classification/solutions/2_keras_for_text_classification.ipynb

```diff
@@ -910,6 +910,218 @@
     "cnn_model.summary()"
```

Comment:
<p>Similarly here. I changed PATIENCE=2 and modified the learning rate to be 0.0001 at compile step setting <code>optimizer=tf.keras.optimizers.Adam(learning_rate=0.0001)</code>. Takes a bit longer to train but much nicer train/validation loss curves.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>Done!</p>